src/sql/workbench/services/tableDesigner/browser/tableDesignerComponentInput.ts

```diff
@@ -53,6 +59,7 @@ export class TableDesignerComponentInput implements DesignerComponentInput {
 		@IInstantiationService private readonly _instantiationService: IInstantiationService,
 		@IErrorMessageService private readonly _errorMessageService: IErrorMessageService) {
 	}
+	designerUIState?: DesignerUIState;
```

Comment:
what is this for? Maybe I'm missing where it gets used

Reply:
weird, let me check, I don't remember adding this.

Reply:
i see, it is added automatically when implementing the interface. it should be kept here. I'll modify it a little bit.